### ISSUE TITLE
20.0a1 type object 'CallbackContext' has no attribute -> DEFAULT_TYPE 

### ISSUE THREAD

**Issue by CarloCattano** [python-telegram-bot/python-telegram-bot#3101]
### Steps to Reproduce

1. Run group member bot from examples 
2. Start the bot 
3. Error ```async def track_chats(update: Update, context: CallbackContext.DEFAULT_TYPE) -> None:
AttributeError: type object 'CallbackContext' has no attribute 'DEFAULT_TYPE' ```

### Expected behaviour

Should run as in previous -pre without issues 

### Actual behaviour

Error and exit

### Operating System

linux

### Version of Python, python-telegram-bot & dependencies

```shell
: python-telegram-bot
Version: 20.0a1
Summary: We have made you a wrapper you can't refuse
Home-page: https://python-telegram-bot.org/
Author: Leandro Toledo
Author-email: devs@python-telegram-bot.org
License: LGPLv3
Location: /home/alarm/dev/bots/unit/professor-moon-home/venv/lib/python3.10/site-packages
Requires: APScheduler, cachetools, httpx, pytz, tornado
Required-by: 

Python 3.10.5
```

**Comment by CarloCattano** [comment]
Ok found the pinned post in telegram , would be nice to have the release notes somewhere on the readme as well for the future ;-)

**Comment by Bibo-Joshi** [comment]
> would be nice to have the release notes somewhere on the readme

Added in https://github.com/python-telegram-bot/python-telegram-bot/pull/3103/commits/cdf9fd6ecc7da51ee3a8bb7e1df382d3c186045a - see #3103
